src/test/java/com/wikia/webdriver/common/core/url/UrlChecker.java

```diff
@@ -25,14 +25,19 @@ public static void isPathContainedInCurrentUrl(WebDriver driver, String path) {
    * This method checks that URL equals current URL and logs result
    * The method is case-insensitive 
    */
-  public static void isUrlEqualToCurrentUrl(WebDriver driver, String url) {
+  public static void assertUrlEqualToCurrentUrl(WebDriver driver, String url) {
     String currentUrl = driver.getCurrentUrl().toLowerCase();
-    url = url.toLowerCase();
     PageObjectLogging.log(
         "Log Url",
         "Url " + url + " is equal to current Url " + currentUrl,
         "Url " + url + " isn't equal to current Url " + currentUrl,
-        currentUrl.equals(url)
-    );
+        isUrlEqualToCurrentUrl(driver, url)
+     );
+  }
+
+  public static boolean isUrlEqualToCurrentUrl(WebDriver driver, String url) {
+    String currentUrl = driver.getCurrentUrl().toLowerCase();
+    url = url.toLowerCase();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "url". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1226)